.github/workflows/rerun-failed.yml

```diff
@@ -0,0 +1,25 @@
+name: Rerun failed
+
+on:
+  workflow_run:
+    workflows: [Run]
+    types: [completed]
```

Comment:
This ensures that we can safely restart the workflow, as it fires when the entire run completes (could be in a failed state)